### PR TITLE
RESFACIL-434: re-organized past training materials

### PR DESCRIPTION
**Pull request by rynge**
- Moved past materials to their own page
- Grouped into general, data, software, workflows, special environments
- Added embedded youtube videos for the trainings with videos

### CODE REVIEW

**Review by ChristinaLK — APPROVED** [review]
this looks great! and aligns with some conversations that Amber and I had about rearranging content (I think).

File: data/configs/ospool.yml
Comment (aowen-uwmad) [inline data/configs/ospool.yml:1]: LGTM

File: documentation/support_and_training/training/materials.md
Comment (aowen-uwmad) [inline documentation/support_and_training/training/materials.md:1]: If just a reordering of what we had on the training page previously, should be fine.
May want to double check that everything renders as expected after merge.

File: documentation/support_and_training/training/osgusertraining.md
Comment (aowen-uwmad) [inline documentation/support_and_training/training/osgusertraining.md:1]: LGTM

**Review by aowen-uwmad — APPROVED** [review]
Looks good. We should check that it renders as expected.

**Comment by aowen-uwmad** [comment]
Confirmed that things rendered correctly.

**Pull request merged**